### PR TITLE
Add verbose_progress argument to control_bayes()

### PR DESCRIPTION
**Pull request by EmilHvitfeldt**
Related to https://github.com/tidymodels/finetune/issues/44.

### CODE REVIEW

File: R/tune_bayes.R [hunk 1]
```diff
@@ -618,12 +618,7 @@ more_results <- function(object, resamples, candidates, metrics, control, param_
         param_info = param_info,
         grid = candidates,
         metrics = metrics,
-        control = control_grid(
-          verbose = FALSE,
-          extract = control$extract,
-          save_pred = control$save_pred,
-          event_level = control$event_level
-        )
+        control = parsnip::condense_control(control, control_grid())
```
Comment (EmilHvitfeldt) [inline R/tune_bayes.R:621]: This will pass more argument through then previously, but it seems like the right thing to do

Comment (EmilHvitfeldt) [reply R/tune_bayes.R]: I realized that we don't even need this as `tune_grid()` does this under the hood

**Comment by EmilHvitfeldt** [comment]
@topepo once this PR is approved, then I'll make the corresponding finetune PR

**Comment by topepo** [comment]
Can you add some testing snapshots for a small problem?

**Pull request merged**